### PR TITLE
K8SPG-315 Incorporate pg v2 images

### PR DESCRIPTION
**Pull request by cap1984**
PG v2 is not the latest operator version available due to version branching, so, no reflection inside latest operator section tests

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | failed |

File: sources/operator.2.0.0.pg-operator.dep.json [hunk 1]
```diff
@@ -0,0 +1,202 @@
+{
+  "pgbackrest":{
+     "14.6":{
```
Comment (hors) [inline sources/operator.2.0.0.pg-operator.dep.json:3]: we have PG 15 for v2

File: sources/operator.2.0.0.pg-operator.dep.json [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+                 {
+                    "var":"productVersion"
+                 },
+                 "13.0"
+              ]
+           }
+        ]
+     }
+  },
+  "pgbackrest_repo":{
```
Comment (hors) [inline sources/operator.2.0.0.pg-operator.dep.json:52]: we do not have pgbackrest image

File: sources/operator.2.0.0.pg-operator.json [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+            "status": "recommended",
+            "critical": false
+          },
+          "13.9": {
+            "image_path": "percona/percona-postgresql-operator:2.0.0-ppg13-postgres",
+            "image_hash": "0a36fd94c0686b1cb8c0fecd30c31fa609a00ab127a7cb5ba76c1c9300843065",
+            "status": "recommended",
+            "critical": false
+          },
+          "14.6": {
```
Comment (hors) [inline sources/operator.2.0.0.pg-operator.json:20]: please add 15.2

**Review by hors — CHANGES_REQUESTED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

**Review by egegunes — APPROVED** [review]

**Review by hors — APPROVED** [review]

**Pull request merged**